src/game_battlealgorithm.h

```diff
@@ -363,6 +363,7 @@ class AlgorithmBase {
 	std::string GetDamagedMessage() const;
 	std::string GetParameterChangeMessage(bool is_positive, int value, const std::string& points) const;
 	std::string GetStateMessage(const std::string& message) const;
+	std::string GetShiftAttributeMessage(bool is_positive, const std::string& attribute) const;
```

Comment:
The is\_positive arg is useless, GetShiftAttribMessage itself could simply call IsPositive